plugins/quote.py

```diff
@@ -146,6 +146,58 @@ def get_quote_by_chan(db, chan, num=False):
     return format_quote(data, num, count)
 
 
+find_re = re.compile("(?:(#\S+)\s+)?(\S+)\s+(.*)$")
```

Comment:
Either replace `\S` and the like with `\\S` or use a raw string to avoid invalid escapes.